app/models/post.rb

```diff
@@ -0,0 +1,21 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  has_many :coments, dependent: :destroy
+
+  validates :title, presence: true
+  validates :text, presence: true
+  validates :comments_counter, presence: true
+  validates :likes_counter, presence: true
+
+  after_create :increase_posts_counter
+
+  def recent_comments
+    coments.order(created_at: :desc).limit(5)
+  end
+
+  def increase_posts_counter
+    self.post_counter += 1
+    save
+  end
```

Comment:
- [ ] I got an error indicating that the `post_counter` attribute is not defined. Kindly fix this issue before resubmitting it for review. I would advise you to use the `increment` Ruby method to increment counters.

```ruby
def increase_posts_counter
  author.increment!(:post_counter)
end
```


![model](https://github.com/MoyasiGinko/Rails-Blog/assets/53124548/01fe9bbd-6999-4721-92e2-135e58814830)